public/index.html

```diff
@@ -21,11 +21,9 @@
       crossorigin="anonymous"
     ></script>
     <!-- Latest compiled and minified JavaScript -->
-    <script
-      src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"
-      integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa"
-      crossorigin="anonymous"
-    ></script>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
```

Comment:
https://getbootstrap.com/docs/4.5/getting-started/introduction/#js

Reply:
@jay-hodgson as a side note, https://github.com/Sage-Bionetworks/Synapse-React-Client/blob/develop/src/demo/SingleFileBuild/index.html uses
```
  <script
    src="https://unpkg.com/react-bootstrap@next/dist/react-bootstrap.min.js"
    crossorigin
  ></script>
```
I'm not sure if this needs to be changed or not.

Reply:
Yes, I think so.  It currently redirects to 1.1.0-rc.0:
https://unpkg.com/react-bootstrap@1.1.0-rc.0/dist/react-bootstrap.min.js

It should probably be replaced with 1.3.0:
https://unpkg.com/react-bootstrap@1.3.0/dist/react-bootstrap.min.js